dp-cluster-setup-utility.py

```diff
@@ -420,7 +424,18 @@ def __init__(self, client, a_dict):
     self.component = a_dict
 
   def host_names(self):
-    return [each.hostname for each in self.component.host_ref]
+    host_id = self.component.host_ref.host_id
```

Comment:
This seems to change behavior. For e.g. the previous code was returning an array and this one is returning a single response. Any reason?

Reply:
yes, since we were not using that feature it was not validated. The response contains an object and not array of objects 
``` 'host_ref': {'host_id': 'cb8c5334-c487-4740-b6cf-831ab96d8ac3',
                         .... }, 
```